the-collab-lab/tcl-79-smart-shopping-list
# Replaced dropdown with radio buttons in AddItemForm.

## Description

Replaced the `daysUntilNextPurchase` dropdown with radio buttons in the AddItemForm modal for improved selection.

## Related Issue

closes #46 

## Acceptance Criteria

- [x] Replace the dropdown with radio buttons on the AddItemForm modal. 

## Type of Changes

Enhancement

## Updates

### Before

![Screen Shot 2024-10-03 at 2 05 25 PM](https://github.com/user-attachments/assets/3333887a-f50e-4255-80d0-647bedff04ec)

### After

![Screen Shot 2024-10-09 at 3 06 01 PM](https://github.com/user-attachments/assets/b61d3671-b741-4394-ae42-abb269a9bafa)


## Testing Steps / QA Criteria

- Run `git pull` and checkout the branch `hm-addItemForm-relocation-and-modifications`.
- Open the homepage by running npm start.
- Navigate to the list page and click on the the "+" button to display the add modal. 
- You can see the new radio buttons which replaced the dropdown for `daysUntilNextPurchase`.
